2018-04-13 01:46:33.146 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 01:46:33.925 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 01:47:38.476 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 01:47:39.133 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 01:47:45.620 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'hbulens.ServiceBus.Publisher.Message, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: 28f29903-eaf0-4381-90ab-a8c00188251c
2018-04-13 01:47:45.674 ERROR NServiceBus.RecoverabilityExecutor Moving message '28f29903-eaf0-4381-90ab-a8c00188251c' to the error queue 'error' because processing failed due to an exception:
NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message 28f29903-eaf0-4381-90ab-a8c00188251c ---> System.Exception: Could not determine type for node: 'Message'.
   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 210
   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 202
   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 56
   at NServiceBus.DeserializeLogicalMessagesConnector.Extract(IncomingMessage physicalMessage) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 118
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 46
   --- End of inner exception stack trace ---
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 50
   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.UnitOfWorkBehavior.<InvokeUnitsOfWork>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\UnitOfWork\UnitOfWorkBehavior.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MutateIncomingTransportMessageBehavior.<InvokeIncomingTransportMessagesMutators>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\MessageMutators\MutateTransportMessage\MutateIncomingTransportMessageBehavior.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Performance\Statistics\ProcessingStatisticsBehavior.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 39
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\MainPipelineExecutor.cs:line 34
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Transports\Learning\LearningTransportMessagePump.cs:line 274
2018-04-13 02:12:38.918 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:12:39.542 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 02:12:45.201 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'Sibusiso.ServiceBus.Publisher.Message, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: d91966ff-6335-42a7-ab40-a8c100038076
2018-04-13 02:15:21.596 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:15:22.194 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 02:15:22.634 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'Sibusiso.ServiceBus.Publisher.Message, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: d91966ff-6335-42a7-ab40-a8c100038076
2018-04-13 02:15:22.677 ERROR NServiceBus.RecoverabilityExecutor Moving message 'd91966ff-6335-42a7-ab40-a8c100038076' to the error queue 'error' because processing failed due to an exception:
NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message d91966ff-6335-42a7-ab40-a8c100038076 ---> System.Exception: Could not determine type for node: 'Message'.
   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 210
   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 202
   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 56
   at NServiceBus.DeserializeLogicalMessagesConnector.Extract(IncomingMessage physicalMessage) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 118
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 46
   --- End of inner exception stack trace ---
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 50
   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.UnitOfWorkBehavior.<InvokeUnitsOfWork>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\UnitOfWork\UnitOfWorkBehavior.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MutateIncomingTransportMessageBehavior.<InvokeIncomingTransportMessagesMutators>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\MessageMutators\MutateTransportMessage\MutateIncomingTransportMessageBehavior.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Performance\Statistics\ProcessingStatisticsBehavior.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 39
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\MainPipelineExecutor.cs:line 34
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Transports\Learning\LearningTransportMessagePump.cs:line 274
2018-04-13 02:17:46.415 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:17:47.125 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 02:17:52.046 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'Sibusiso.ServiceBus.Publisher.Message, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: f32b3495-99b7-4a3e-8428-a8c10004e80a
2018-04-13 02:17:52.091 ERROR NServiceBus.RecoverabilityExecutor Moving message 'f32b3495-99b7-4a3e-8428-a8c10004e80a' to the error queue 'error' because processing failed due to an exception:
NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message f32b3495-99b7-4a3e-8428-a8c10004e80a ---> System.Exception: Could not determine type for node: 'Message'.
   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 210
   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 202
   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 56
   at NServiceBus.DeserializeLogicalMessagesConnector.Extract(IncomingMessage physicalMessage) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 118
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 46
   --- End of inner exception stack trace ---
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 50
   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.UnitOfWorkBehavior.<InvokeUnitsOfWork>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\UnitOfWork\UnitOfWorkBehavior.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MutateIncomingTransportMessageBehavior.<InvokeIncomingTransportMessagesMutators>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\MessageMutators\MutateTransportMessage\MutateIncomingTransportMessageBehavior.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Performance\Statistics\ProcessingStatisticsBehavior.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 39
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\MainPipelineExecutor.cs:line 34
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Transports\Learning\LearningTransportMessagePump.cs:line 274
2018-04-13 02:19:08.134 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:19:08.450 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 02:19:20.656 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'Sibusiso.ServiceBus.Publisher.Message, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: 48d8db25-c9b4-4f00-bde6-a8c100054fdd
2018-04-13 02:19:20.701 ERROR NServiceBus.RecoverabilityExecutor Moving message '48d8db25-c9b4-4f00-bde6-a8c100054fdd' to the error queue 'error' because processing failed due to an exception:
NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message 48d8db25-c9b4-4f00-bde6-a8c100054fdd ---> System.Exception: Could not determine type for node: 'Message'.
   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 210
   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 202
   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 56
   at NServiceBus.DeserializeLogicalMessagesConnector.Extract(IncomingMessage physicalMessage) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 118
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 46
   --- End of inner exception stack trace ---
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 50
   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.UnitOfWorkBehavior.<InvokeUnitsOfWork>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\UnitOfWork\UnitOfWorkBehavior.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MutateIncomingTransportMessageBehavior.<InvokeIncomingTransportMessagesMutators>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\MessageMutators\MutateTransportMessage\MutateIncomingTransportMessageBehavior.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Performance\Statistics\ProcessingStatisticsBehavior.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 39
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\MainPipelineExecutor.cs:line 34
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Transports\Learning\LearningTransportMessagePump.cs:line 274
2018-04-13 02:20:56.312 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:20:56.649 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 02:21:05.863 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'Sibusiso.ServiceBus.Publisher.Message, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: d272240f-06c8-498f-a4fe-a8c10005cb27
2018-04-13 02:21:05.909 ERROR NServiceBus.RecoverabilityExecutor Moving message 'd272240f-06c8-498f-a4fe-a8c10005cb27' to the error queue 'error' because processing failed due to an exception:
NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message d272240f-06c8-498f-a4fe-a8c10005cb27 ---> System.Exception: Could not determine type for node: 'Message'.
   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 210
   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 202
   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 56
   at NServiceBus.DeserializeLogicalMessagesConnector.Extract(IncomingMessage physicalMessage) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 118
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 46
   --- End of inner exception stack trace ---
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 50
   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.UnitOfWorkBehavior.<InvokeUnitsOfWork>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\UnitOfWork\UnitOfWorkBehavior.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MutateIncomingTransportMessageBehavior.<InvokeIncomingTransportMessagesMutators>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\MessageMutators\MutateTransportMessage\MutateIncomingTransportMessageBehavior.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Performance\Statistics\ProcessingStatisticsBehavior.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 39
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\MainPipelineExecutor.cs:line 34
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Transports\Learning\LearningTransportMessagePump.cs:line 274
2018-04-13 02:22:42.813 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:22:43.708 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 02:22:50.269 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'Sibusiso.ServiceBus.Publisher.ServiceMessage, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: 60bca25b-d7db-4dab-ab28-a8c10006458d
2018-04-13 02:22:50.315 ERROR NServiceBus.RecoverabilityExecutor Moving message '60bca25b-d7db-4dab-ab28-a8c10006458d' to the error queue 'error' because processing failed due to an exception:
NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message 60bca25b-d7db-4dab-ab28-a8c10006458d ---> System.Exception: Could not determine type for node: 'ServiceMessage'.
   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 210
   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 202
   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 56
   at NServiceBus.DeserializeLogicalMessagesConnector.Extract(IncomingMessage physicalMessage) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 118
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 46
   --- End of inner exception stack trace ---
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 50
   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.UnitOfWorkBehavior.<InvokeUnitsOfWork>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\UnitOfWork\UnitOfWorkBehavior.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MutateIncomingTransportMessageBehavior.<InvokeIncomingTransportMessagesMutators>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\MessageMutators\MutateTransportMessage\MutateIncomingTransportMessageBehavior.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Performance\Statistics\ProcessingStatisticsBehavior.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 39
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\MainPipelineExecutor.cs:line 34
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Transports\Learning\LearningTransportMessagePump.cs:line 274
2018-04-13 02:24:22.017 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:24:22.462 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 02:24:29.342 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'Sibusiso.ServiceBus.Publisher.ServiceMessage, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: 9b9c781c-80c7-4839-bf6a-a8c10006b99d
2018-04-13 02:24:29.398 ERROR NServiceBus.RecoverabilityExecutor Moving message '9b9c781c-80c7-4839-bf6a-a8c10006b99d' to the error queue 'error' because processing failed due to an exception:
NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message 9b9c781c-80c7-4839-bf6a-a8c10006b99d ---> System.Exception: Could not determine type for node: 'ServiceMessage'.
   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 210
   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 202
   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 56
   at NServiceBus.DeserializeLogicalMessagesConnector.Extract(IncomingMessage physicalMessage) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 118
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 46
   --- End of inner exception stack trace ---
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 50
   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.UnitOfWorkBehavior.<InvokeUnitsOfWork>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\UnitOfWork\UnitOfWorkBehavior.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MutateIncomingTransportMessageBehavior.<InvokeIncomingTransportMessagesMutators>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\MessageMutators\MutateTransportMessage\MutateIncomingTransportMessageBehavior.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Performance\Statistics\ProcessingStatisticsBehavior.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 39
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\MainPipelineExecutor.cs:line 34
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Transports\Learning\LearningTransportMessagePump.cs:line 274
2018-04-13 02:25:48.140 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:25:48.806 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 02:25:52.433 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'Sibusiso.ServiceBus.Publisher.ServiceMessage, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: 69029e75-81cb-4623-84d6-a8c100071af9
2018-04-13 02:25:52.487 ERROR NServiceBus.RecoverabilityExecutor Moving message '69029e75-81cb-4623-84d6-a8c100071af9' to the error queue 'error' because processing failed due to an exception:
NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message 69029e75-81cb-4623-84d6-a8c100071af9 ---> System.Exception: Could not determine type for node: 'ServiceMessage'.
   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 210
   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 202
   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 56
   at NServiceBus.DeserializeLogicalMessagesConnector.Extract(IncomingMessage physicalMessage) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 118
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 46
   --- End of inner exception stack trace ---
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 50
   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.UnitOfWorkBehavior.<InvokeUnitsOfWork>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\UnitOfWork\UnitOfWorkBehavior.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MutateIncomingTransportMessageBehavior.<InvokeIncomingTransportMessagesMutators>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\MessageMutators\MutateTransportMessage\MutateIncomingTransportMessageBehavior.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Performance\Statistics\ProcessingStatisticsBehavior.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 39
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\MainPipelineExecutor.cs:line 34
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Transports\Learning\LearningTransportMessagePump.cs:line 274
2018-04-13 02:28:18.378 INFO  DefaultFactory Logging to 'c:\users\freedom.nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:28:19.060 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

2018-04-13 02:28:24.591 WARN  NServiceBus.DeserializeLogicalMessagesConnector Could not determine message type from message header 'Sibusiso.ServiceBus.Publisher.ServiceMessage, Sibusiso.ServiceBus.Publisher, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. MessageId: 5f94ecaa-d4a2-496b-881f-a8c10007cd46
2018-04-13 02:28:24.639 ERROR NServiceBus.RecoverabilityExecutor Moving message '5f94ecaa-d4a2-496b-881f-a8c10007cd46' to the error queue 'error' because processing failed due to an exception:
NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message 5f94ecaa-d4a2-496b-881f-a8c10007cd46 ---> System.Exception: Could not determine type for node: 'ServiceMessage'.
   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 210
   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 202
   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Serializers\XML\XmlDeserialization.cs:line 56
   at NServiceBus.DeserializeLogicalMessagesConnector.Extract(IncomingMessage physicalMessage) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 118
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 46
   --- End of inner exception stack trace ---
   at NServiceBus.DeserializeLogicalMessagesConnector.ExtractWithExceptionHandling(IncomingMessage message) in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 50
   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\DeserializeLogicalMessagesConnector.cs:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.UnitOfWorkBehavior.<InvokeUnitsOfWork>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\UnitOfWork\UnitOfWorkBehavior.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MutateIncomingTransportMessageBehavior.<InvokeIncomingTransportMessagesMutators>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\MessageMutators\MutateTransportMessage\MutateIncomingTransportMessageBehavior.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Performance\Statistics\ProcessingStatisticsBehavior.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\Incoming\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 39
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Pipeline\MainPipelineExecutor.cs:line 34
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\BuildAgent\work\b549d46003942065\src\NServiceBus.Core\Transports\Learning\LearningTransportMessagePump.cs:line 274
2018-04-13 02:45:33.338 INFO  DefaultFactory Logging to 'C:\Users\Freedom.Nxumalo\source\repos\Sibusiso.ContactDetails.Service\Sibusiso.ServiceBus.Reciever\bin\Debug\netcoreapp2.0\' with level Info
2018-04-13 02:45:33.901 INFO  NServiceBus.LicenseManager No valid license could be found. Falling back to trial license with start date '2018-04-12'.

